addons/test_event_full/tests/test_performance.py

```diff
@@ -452,7 +452,7 @@ def test_events_browse_public(self):
         # website customer data
         with freeze_time(self.reference_now):
             self.authenticate(None, None)
-            with self.assertQueryCount(default=32):  # tef only: 32
+            with self.assertQueryCount(default=33):  # tef only: 33
```

Comment:
Do you know where this extra query comes from?
If yes, would be nice to add a small note about it in the commit message as well.
So it looks like this is a controlled and intended change.

Reply:
Apparently it comes from the changes in `event_templates_list`. I don't know why but when I remove all the `with_context()` part there is an extra query. So to avoid this extra query, I've put back this part since it doesn't change anything functionally